go.mod

```diff
@@ -3,19 +3,19 @@ module github.com/celestiaorg/test-infra
 go 1.19
 
 replace (
-	github.com/celestiaorg/celestia-app => github.com/Bidon15/lazyledger-app v1.12.1-dirty-app
-	github.com/celestiaorg/celestia-node => github.com/Bidon15/celestia-node v0.7.1-rc2-dirty
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.6.0-sdk-v0.46.7
+	github.com/celestiaorg/celestia-node => github.com/derrandz/celestia-node v0.4.2-0.20230202230649-5c4e0c179d7c
```

Comment:
Can you cut a release on your fork(v0.7.0-black-box-metrics)? 🙏 
last thing we want is to be confused with the current way we see release numbers